frontend/public/components/modals/configure-ns-pull-secret-modal.jsx

```diff
@@ -207,7 +207,7 @@ class ConfigureNamespacePullSecretWithTranslation extends PromiseComponent {
               title={t('public~Overwriting default pull Secret')}
             >
               {t(
-                'public~A default pull Secret exists, but can&apos;t be parsed. Saving this will overwrite it.',
+                "public~A default pull Secret exists, but can't be parsed. Saving this will overwrite it.",
```

Comment:
I think we should be able to remove any logic around editing an existing pull secret from this modal since we're changing it to a link. The dialog should only be used for new pull secrets now, correct?

Reply:
Good catch, removed code for displaying existing secrets within the modal.

Reply:
Note.  I'll squash commits after LGTM approval.